pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "monkeypox"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
I usually put info@global.health here - though that may not be ideal as I'm not sure if we have access to that